src/objects.js

```diff
@@ -1,41 +1,64 @@
 const createPerson = (name, age) => {
-  // your code here
+  return { name, age };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  if (object[property]) return true;
+  return false;
 };
 
 const isOver65 = person => {
-  // your code here
+  if (person.age > 65) return true;
```

Comment:
This is great, you could shorten it though as `person.age > 65` already returns a boolean value. eg `return person.age > 65`